nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ValidateCsv.java

```diff
@@ -142,6 +142,21 @@ public class ValidateCsv extends AbstractProcessor {
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .build();
 
+    public static final PropertyDescriptor MAX_LINES_PER_ROW = new PropertyDescriptor.Builder()
+            .name("max-lines-per-row")
+            .displayName("Max Lines Per Row")
```

Comment:
```suggestion
            .name("Max Lines Per Row")
```

`.displayName` should not be used for newly added property

Reply:
You mean to remove it entirely?

Reply:
.displayName removed